jac-uk/admin
# #1077 Display responses provided in a previous test

## What's included?
This PR fixes the issue of not being able to see the answers a candidate provided previously by showing the questions and answers all the time (if they present), even if the test is not completed.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
You will need to start or complete answering a qualifying test response (or locate an existing one). Previously if this test was reset or moved to mop up it would not be possible to see the answers already provided. Move your test to a mop up date and observe that the answers already provided can be seen.

For speed, here is an existing response which has been moved to mop up:
https://jac-admin-develop--pr1259-feature-1077-display-7jlvq929.web.app/exercises/wdpALbyICL7ZxxN5AQt8/tasks/qualifying-tests/JTx6phqRwUJwroLgkBgR/response/Ls2eoLaM56Lgg1RrXFJb/
(should be able to see the answers provided so far)

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
See @mbrookeswebdev comment & screen grab below for more info around testing

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
